### PR TITLE
Add IHttpResponseWriter documentation

### PR DESCRIPTION
**Pull request by josephdecock**

### CODE REVIEW

**Review by brockallen — APPROVED** [review]

**Comment by brockallen** [comment]
LGTM -- not sure if you have any last updated before we merge?

**Comment by josephdecock** [comment]
All done - just noticed the namespaces were inconsistent

**Pull request merged**
